mathesar_ui/src/component-library/icon/Icon.svelte

```diff
@@ -68,6 +71,9 @@
   role={label ? 'img' : 'presentation'}
   {...$$restProps}
 >
+  {#if title}
+    <title>{title}</title>
+  {/if}
```

Comment:
I'm okay with using the title property as standby icon tooltips, but the actual usecase requires us to have a tooltip component, which isn't reserved to icons.

Please create a separate issue for this.

Reply:
https://github.com/centerofci/mathesar/issues/1982